webhook_to_fedora_messaging/main.py

```diff
@@ -15,7 +15,7 @@
 from .config.defaults import LOGGER_CONFIG
 
 from logging.config import dictConfig
-
+from .endpoints.user import user_endpoint
```

Comment:
Ref. https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/24#discussion_r1667901072